projects/QuotesApplication/app/src/main/java/com/quotesapplication/MainActivity.java

```diff
@@ -0,0 +1,98 @@
+package com.quotesapplication;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.NotificationCompat;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class MainActivity extends AppCompatActivity {
+    Button btNotification;
+    Button btTimingNotification;
+    static String CHANNEL_ID = "CHANNEL";
+    public static String[] quotesArray;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        //Call to createQuotesArray and throw exception if there is any problem with the file
+        try {
+            createQuotesArray();
+        } catch (IOException exception) {
+            exception.printStackTrace();
+        }
+        createNotificationChannel();
+
+        //OnClick function for immediate quote notification
+        btNotification = (Button)findViewById(R.id.quotesButtonNow);
+        btNotification.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String quote = quotesArray[(int)(Math.random() * 60)];
```

Comment:
use quotesArray.length instead of 60, btw, Math.random() * 60? can get an out of index exception.